tests/test_schemas.py

```diff
@@ -30,6 +30,9 @@ def test_library_schema_full(self):
         ):
             excluded_points.append(("/debugger/v1/input", "$[].debugger.snapshot.stack[].lineNumber"))
 
+        if context.library == "nodejs":
+            excluded_points.append(("/debugger/v1/diagnostics", "$[].content[].debugger.diagnostics"))  # DEBUG-3245
```

Comment:
The only error we want to ignore is if `probeVersion` is missing on `$[].content[].debugger.diagnostics`, but I couldn't find a way to do that without excluding all errors on `$[].content[].debugger.diagnostics`.

And technically we only want to ignore the error for certain mocked RC responses that doesn't contain a `version` property, but I don't think that's possible.

Reply:
Unfortunately, the schema test is not powerful enough to do that precision. The solution is  totally ignore `$[].content[].debugger.diagnostics` in schema, and write a python test for thta